datadog/measure.go

```diff
@@ -3,20 +3,28 @@ package datadog
 import (
 	"math"
 	"strconv"
+	"strings"
 
 	"github.com/segmentio/stats/v4"
 )
 
+// Datagram format: https://docs.datadoghq.com/developers/dogstatsd/datagram_shell
+
 // AppendMeasure is a formatting routine to append the dogstatsd protocol
 // representation of a measure to a memory buffer.
 func AppendMeasure(b []byte, m stats.Measure) []byte {
-	return AppendMeasureFiltered(b, m, nil)
+	return appendMeasureDist(b, m, []string{})
+}
+
+func appendMeasureDist(b []byte, m stats.Measure, distPrefixes []string) []byte {
+	return AppendMeasureFiltered(b, m, nil, distPrefixes)
```

Comment:
This function looks un-necessary, right ?

Reply:
You're right, it can be removed. Done!